frontend/public/locales/en/translation.json

```diff
@@ -1073,7 +1072,6 @@
   "Decisions": "Decisions",
   "decoupled control plane and workers": "decoupled control plane and workers",
   "dedicated control plane nodes": "dedicated control plane nodes",
-  "Default OpenShift": "Default OpenShift",
```

Comment:
John and I had a discussion about this and we thought users might be confused by this filter since it actually never appears on the Application table Type column. Maybe we need to word it better?

Reply:
Introduced "System" category in new PR https://github.com/stolostron/console/pull/3753